src/auth.c

```diff
@@ -1716,13 +1716,16 @@ int pr_auth_chroot(const char *path) {
 
   tz = pr_env_get(session.pool, "TZ"); 
   if (tz == NULL) {
+    char tzbuf[32];
+    long off = (timezone / 60 / 60) - daylight;
+    snprintf(tzbuf, sizeof(tzbuf), "%s%ld", daylight ? tzname[1] : tzname[0], off);
```

Comment:
Well, since this code path would be executed before the chroot(2) call, the /etc/localtime file would need to be accessible the first time any ctime(3) or related functions were called.  Which happens, I think, because of the pr_log_debug() function call, right after the setting of the TZ environment variable.  The logging routines want to add a timestamp to the generated log message, so they call e.g. ctime(3), which in turn looks up TZ...and its loaded.